pydicom/tests/test_filereader.py

```diff
@@ -933,7 +937,15 @@ def test_seq_item_looks_like_explicit_VR(self):
         ds.remove_private_tags()  # forces it to actually parse SQ
 
 
+
 class TestUnknownVR:
+    @pytest.fixture(autouse=True)
+    def restore_config_values(self):
```

Comment:
The fixture name is a bit strange - it actually changes the config value (restoring it afterwards is assumed in any fixture).

Reply:
Yeah, I agree - but I just borrowed the name from an existing test class.